pkg/crc/validation/validation.go

```diff
@@ -101,6 +114,39 @@ func ValidateIPAddress(ipAddress string) error {
 	return nil
 }
 
+func ValidateURL(uri string) error {
+	u, err := url.ParseRequestURI(uri)
+	if err != nil {
+		logging.Debugf("Failed to parse url: %v", err)
+		return err
+	}
+	// If uri string is without scheme then check if it is valid path
+	if !u.IsAbs() {
+		return ValidatePath(uri)
```

Comment:
https://go.dev/play/p/oCboAo7HPBI as per `url.ParseRequestURI()` it consider `/tmp/bundle` as valid URI but `bundle` as invalid so with `u.IsAbs()` I can check if is abs path and then validate the path but in case user have bundle at the same location as crc binary and don't use absolute path then URI validation fails and I have to again check it with `ValidatePath`. With current logic both required.

Reply:
I'd add this to the comment 
```
// If uri string is without scheme then check if it is a valid absolute path.
// Relative paths will cause `ParseRequestURI` to error out, and will be handled in `ValidateBundlePath`
```

What about Windows paths, `C:\bundles\...`?